.github/actions/release-branch/action.yaml

```
@@ -0,0 +1,86 @@
+name: "Create Release Branch from Tag"
+description: "Creates a release branch from any tag"
+
+inputs:
+  branch-prefix:
+    description: "Prefix used for branch names"
+    required: false
+    default: "release-"
```

```suggestion
    description: "Generated branch name will be of form {prefix}{component}/v{major.minor}"
    required: true
```
Optionally: always add a `/` after prefix and change caller to just set this to `release`